packages/features/eventtypes/components/CreateEventTypeDialog.tsx

```diff
@@ -136,6 +139,9 @@ export default function CreateEventTypeDialog({
   });
 
   const flags = useFlagMap();
+  const urlPrefix = orgBranding
+    ? `${orgBranding.slug}.${subdomainSuffix()}`
+    : process.env.NEXT_PUBLIC_WEBSITE_URL;
```

Comment:
Abstracting the calculation of the URL to be used in the dialog

Reply:
NIT/Future work: 
We do this in a few places across the whole of ORGs

Do you think we should extract this? 

Instead of a specific `useOrgBranding` context in the other PR maybe we can just introduce a `useOrg` context that has things like this without needing to do this/calculate username url etc every time

Reply:
Created CAL-1936